views/plugin-page.php

```diff
@@ -197,3 +201,10 @@
 
 	<div class="ocdi__response  js-ocdi-ajax-response"></div>
 </div>
+
+<?php
+/**
+ * Hook for adding the custom admin page footer
+ */
+do_action('ocdi_admin_page_footer');
+?>
```

Comment:
Last `?>` should be omitted and file should end with the line break.

Also, please follow the WP codex for code style which adds space character inside brackets, like so:

```
do_action( 'ocdi_admin_page_footer' );
```

Reply:
Oh, and also use the same format for hooks as the other hooks are using in the plugin: `pt-ocdi/...`